src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -0,0 +1,83 @@
+# Shard Size Tuning
+
+The purpose of this document is to compare shard size of 8K and 16K ledgers.
```

Comment:
This sentence sounds ambiguous. Consider: _"... is to compare the sizes of shards containing varying amounts of ledgers."_

Reply:
Done.